packages/vite/src/node/server/middlewares/indexHtml.ts

```diff
@@ -268,7 +268,7 @@ const devHtmlHook: IndexHtmlTransformHook = async (
         getScriptInfo(node)
```

Comment:
Maybe we should rename `sourceCodeLocation` returned from `getScriptInfo`. It's actually the location of the `src` attribute.
https://github.com/vitejs/vite/blob/eb06ec30bb02ced66274f0fc6e90aff2bb20c632/packages/vite/src/node/plugins/html.ts#L217

I think `srcSourceCodeLocation` would work.
```ts
const { src, srcSourceCodeLocation, isModule, isIgnored } = getScriptInfo(node)
// in getScriptInfo
return { src, srcSourceCodeLocation, isModule, isAsync, isIgnored }
```

Reply:
Yeah, that makes sense.